### ISSUE TITLE
Piping hooks ouput

### ISSUE THREAD

**Issue by crimson-med** [release-it/release-it#552]
Hello, 

I have been using release-it for a while and i'm really loving it.

I use the hooks extensively to do some ssh connections and code deployment. for instance:

```json
{
  "git": {
    "tagName": "v${version}",
    "pushRepo": "https://github.com/xxxxxxxxxx/yyyyyyyyyy.git"
  },
  "github": {
    "release": true
  },
  "hooks": {
   "before:init": "echo please run: export GITHUB_TOKEN=\"xxxxxxxxxxxxxxxx\"",
   "after:bump": "git push xxxxx master --follow-tags",
   "after:git:release": [
       "ssh -t user@AA.AAA.AA.AAA 'sudo ./pack.sh v${version}'",
       "ssh -t user@AA.AAA.AA.AAA 'sudo ./unpack.sh v${version}'",
       "ssh -t user@AA.AAA.AA.AAA 'sudo ./unpack.sh v${version} && php /var/xxx/xxx/xxx migrate'"],
   "after:release": "echo Successfully released ${name} v${version} to ${repo.repository}.\n"
 }
}
```

My issue is how can I have the output of the ssh commands and pack commands pipe to the ouput of release it. 

Right now I get: 

? Push? Yes
✔ ssh -t user@AA.AAA.AA.AAA 'sudo ./pack.sh v2.2.2'
✔ ssh -t user@AA.AAA.AA.AAA 'sudo ./unpack.sh v2.2.2'
✔ ssh -t user@AA.AAA.AA.AAA 'sudo ./unpack.sh v2.2.2 && php /xxx/xxx/xxx/xxx migrate'
? Create a release on GitHub (Release 2.2.2)? Yes
✔ echo Successfully released xxxx v2.2.2 to xxxx/xxxx.

🔗 xxxx/v2.2.2
🏁 Done (in 224s.)

I would like after each command to also see what the command is echoing.

**Comment by webpro** [comment]
You can use `--verbose` to see this output. However, you will also get the output of the non-custom release-it commands.

**Comment by crimson-med** [comment]
@webpro what do you mean by non-custom commands output as well?

**Comment by webpro** [comment]
I mean the commands you don't configure yourself. E.g. release-it does `git commit` or `npm publish` for you, and `--verbose` gives you the output for this as well.

**Comment by crimson-med** [comment]
@webpro I see would there be any option for just logging the information from custom commands I added?

**Comment by webpro** [comment]
I wish I could say yes (was also requested in #454). PR is of course welcome for this, otherwise I'll take a stab at this when I'll find some time.

**Comment by crimson-med** [comment]
Thanks for the information I will look if I have the time to make a PR.

**Comment by webpro** [comment]
This has just been released in [v12.4.0](https://github.com/release-it/release-it/releases/tag/12.4.0).
